src/DssProxyActionsStETH.t.sol

```diff
@@ -0,0 +1,320 @@
+        uint256 prevBalance = stETH.balanceOf(address(this));
+        this.lockStETHAndDraw(address(jug), address(wstETHJoin), address(daiJoin), cdp, 2 ether, 10 ether);
+        assertEq(ink("WSTETH", manager.urns(cdp)), wstETH.getWstETHByStETH(2 ether));
+        assertEq(dai.balanceOf(address(this)), 10 ether);
+        assertEq(stETH.balanceOf(address(this)), prevBalance - 2 ether);
+    }
+
+    function testOpenLockStETHAndDraw() public {
+        stETH.approve(address(proxy), 2 ether);
+        assertEq(dai.balanceOf(address(this)), 0);
+        uint256 prevBalance = stETH.balanceOf(address(this));
+        uint256 cdp = this.openLockStETHAndDraw(address(jug), address(wstETHJoin), address(daiJoin), "WSTETH", 2 ether, 10 ether);
+        assertEq(ink("WSTETH", manager.urns(cdp)), wstETH.getWstETHByStETH(2 ether));
+        assertEq(dai.balanceOf(address(this)), 10 ether);
+        assertEq(stETH.balanceOf(address(this)), prevBalance - 2 ether);
+    }
+
+    function testWipeAndFreeStETH() public {
+        uint256 cdp = this.open(address(manager), "WSTETH", address(proxy));
+        stETH.approve(address(proxy), 2 ether);
+        this.lockStETHAndDraw(address(jug), address(wstETHJoin), address(daiJoin), cdp, 2 ether, 10 ether);
+        dai.approve(address(proxy), 8 ether);
+        uint256 prevBalance = stETH.balanceOf(address(this));
+        this.wipeAndFreeStETH(address(wstETHJoin), address(daiJoin), cdp, 1.5 ether, 8 ether);
+        // Due to unwrap precision loss an extra 1 wei of ink resides in the vault
+        assertEq(ink("WSTETH", manager.urns(cdp)), wstETH.getWstETHByStETH(0.5 ether) + 1);
+        assertEq(art("WSTETH", manager.urns(cdp)), 2 ether);
+        assertEq(dai.balanceOf(address(this)), 2 ether);
+        // Due to unwrap precision loss the returned stETH is 1 wei less than requested
+        assertEq(stETH.balanceOf(address(this)), prevBalance + 1.5 ether - 1);
+        assertEq(dai.balanceOf(address(proxy), 2 ether);
+    }
+
+    function testWipeAllAndFreeStETH() public {
+        uint256 cdp = this.open(address(manager), "WSTETH", address(proxy));
+        stETH.approve(address(proxy), 2 ether);
+        this.lockStETHAndDraw(address(jug), address(wstETHJoin), address(daiJoin), cdp, 2 ether, 10 ether);
+        dai.approve(address(proxy), 10 ether);
+        uint256 prevBalance = stETH.balanceOf(address(this));
+        this.wipeAllAndFreeStETH(address(wstETHJoin), address(daiJoin), cdp, 1.5 ether);
+        // Due to unwrap precision loss an extra 1 wei of ink resides in the vault
+        assertEq(ink("WSTETH", manager.urns(cdp)), wstETH.getWstETHByStETH(0.5 ether) + 1);
+        assertEq(art("WSTETH", manager.urns(cdp)), 0);
+        assertEq(dai.balanceOf(address(this)), 0);
+        // Due to unwrap precision loss the returned stETH is 1 wei less than requested
+        assertEq(stETH.balanceOf(address(this)), prevBalance + 1.5 ether - 1);
+        assertEq(dai.balanceOf(address(proxy), 0);
```

Comment:
Syntax error here as well, and as I said above I'd remove this line.

Reply:
same here, sorry (line 270)